Documents/Help/Changelog.md

```diff
@@ -3,6 +3,7 @@
 =======
 
 # 2024-10-14 - Build 2410 (Patch 3) - October 2024
+* Resolved [#1820](https://github.com/Krypton-Suite/Standard-Toolkit/issues/1820), When KryptonDataGridView.AutoGenerate is set Winforms columns are used. See the issue for full text.
```

Comment:
This is in thw wroing place, please start a new section for `Patch 4`

Reply:
@Smurf-IV 
Added the section